eve/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+version: 0.2
+
+branches:
+  user/*, dev/*:
```

Comment:
Is it on purpose you only build dev/* and user/* branches for pre-merge?
With this yml eve will *not* build the feature/*, bugfix/* branches of the users.
Also Wall-E's q/* and w/* branches won't be automatically built either.